app/javascript/routes.js

```diff
@@ -0,0 +1,11613 @@
+/**
```

Comment:
This file has been generated by mistake, it is supposed to live in `app/javascript/utils/routes.js` and is ignored by Git. Could you please remove it?